gprofiler/profilers/python.py

```diff
@@ -566,7 +329,7 @@ def __init__(
                 logger.warning("PyPerf is supported only on x86_64, falling back to py-spy")
             python_mode = "pyspy"
 
-        if python_mode in ("auto", "pyperf"):
+        if python_mode in ("auto", "pyperf") and not is_windows():
```

Comment:
Here you implicitly avoid pyperf even if it is explicitly requested. Don't do it implicitly - you can add a `is_windows()` check to the `if` on line 564 - if we're not x86_64 OR we're Windows, then fallback to py-spy; and if pyperf was requested explicitly in that case, log `logger.warning("PyPerf is supported only on Linux x86_64, falling back to py-spy")`